source/content/partials/drupal/prepare-local-environment-no-clone-new.md

```diff
@@ -10,7 +10,7 @@ reviewed: ""
 
 1. Review the documentation on [Git](/guides/git/), [Composer](/guides/composer), and [Terminus](/terminus), and install and configure them on your local computer. Pantheon requires Composer 2, at minimum.
 
-   - Mac users can use [Homebrew](https://brew.sh/) to install Git, Composer, and PHP, along with their required dependencies. **Note:** Terminus 3 should be used for PHP >= 8.0. Restart the shell or terminal environment after entering the following command:
+   - Mac users can use [Homebrew](https://brew.sh/) to install Git, Composer, and PHP, along with their required dependencies. **Note:** Terminus 4 should be used for PHP >= 8.4. Restart the shell or terminal environment after entering the following command:
```

Comment:
```suggestion
   - Mac users can use [Homebrew](https://brew.sh/) to install Git, Composer, and PHP, along with their required dependencies. Restart the shell or terminal environment after entering the following command:
```

Reply:
I think even talking about the terminus version here overcomplicates things. We can talk about terminus versions and their compatibility in the terminus docs, but this is a drupal partial.